algorithms/threading/service_thread_pinner.cpp

```diff
@@ -37,12 +37,13 @@
         #include "tbb/tbb.h"
         #include <tbb/task_arena.h>
         #include <tbb/task_scheduler_observer.h>
-        #include <tbb/atomic.h>
         #include <tbb/task_scheduler_init.h>
         #include <tbb/parallel_reduce.h>
         #include <tbb/blocked_range.h>
         #include <tbb/tick_count.h>
         #include <tbb/scalable_allocator.h>
+        #include "services/daal_atomic_int.h"
+using namespace daal::services;
```

Comment:
If you introduces 'using namespace', it makes sense to remove this namespace across entire file from lines where it explicitly used.